pkg/provider/aggregation_provider.go

```diff
@@ -76,11 +79,11 @@ func (p *aggregationProvider) GetExternalMetric(context context.Context, namespa
 }
 
 func (p *aggregationProvider) ListAllMetrics() []provider.CustomMetricInfo {
-	klog.V(1).Infof("ListAllMetrics()")
+	logger.V(1).Info("ListAllMetrics")
```

Comment:
I'm wondering if we can remove this message, or increase the verbosity level.

Reply:
I think we can remove this one. More generally for `V(1).Info` log , see https://github.com/elastic/elasticsearch-k8s-metrics-adapter/pull/52#discussion_r1316964673.